app/services/frontend_logger.rb

```diff
@@ -37,7 +37,16 @@ def track_event(name, attributes)
   # @param [Proc,Method] callable
   # @return [Hash<Symbol,Object>]
   def hash_from_kwargs(hash, callable)
-    kwargs(callable).index_with { |key| hash[key.to_s] }
+    kwargs(callable).index_with do |key|
+      value = hash[key.to_s]
+      key == :opted_in_to_in_person_proofing ? convert_to_boolean(value) : value
```

Comment:
Can we update this from where we're logging in the frontend to pass a boolean instead of a string?

https://github.com/18F/identity-idp/blob/fdf864d1a4c30b5246b0ee96833dd4a2b080fb14/app/javascript/packs/document-capture.tsx#L83

e.g.

```tsx
    opted_in_to_in_person_proofing: optedInToInPersonProofing === 'true',
```

Reply:
Ah I like this approach. I was looking for where the string was passed in from the front end and didn't find it. Thanks for pointing this out!